packages/react-icons/package.json

```diff
@@ -26,7 +26,7 @@
     "update-snapshots": "just-scripts jest -u"
   },
   "devDependencies": {
-    "@types/enzyme": "3.10.3",
+    "@types/enzyme": "3.10.5",
```

Comment:
If you're going to update the enzyme version it needs to be updated everywhere, not just here (and probably best to do as a separate PR).